modules/readme/Makefile

```diff
@@ -3,6 +3,9 @@ export README_FILE ?= README.md
 export README_YAML ?= README.yaml
 export README_TEMPLATE_FILE ?= $(BUILD_HARNESS_PATH)/templates/README.md
 export README_TEMPLATE_YAML ?= $(BUILD_HARNESS_PATH)/templates/README.yaml
+export USED_DATASOURCE ?= $(if $(datasource),$(datasource),$(shell pwd)/$(README_YAML))
+export DATASOURCE ?= $(if $(datasource),$(datasource),$(README_YAML))
```

Comment:
where is lowercase `datasource` ever defined?

Reply:
for overriding default datasource (`README.yaml` in root folder)
![image](https://user-images.githubusercontent.com/11632146/45751843-e6f4bf00-bc13-11e8-922f-ba67d49371d3.png)

Reply:
@osterman now if no any env passed, behaviour will be as earlier, will be used `README.yaml` from root project folder. 
This env just for possibility to change datasource, it isn't use case?

Reply:
@osterman move datasource definition from makefile, but gomplate doesn't support inline shell execution, 
`export INCLUDES ?= $(file://$(shell pwd)/?type=text/plain)`
still here